changelog/fragments/liveness_readiness_probe_for_operator.yaml

```diff
@@ -0,0 +1,67 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible/Helm-based operators, add Liveness and Readiness probe by default using [sigs.k8s.io/controller-runtime/pkg/healthz#CheckHandler](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/healthz#CheckHandler).
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (Optional) For Ansible/Helm-based operators, add Liveness and Readiness probe
+      body: >
+        New projects built with the tool will have the probes configured by default. The endpoints `/healthz` and
+        `/readyz` are available now in the image based provided.
+
+        You can update your pre-existing project to use them. For that update the Dockerfile to use the latest
+        release base image and then, add in the `config/default/manager`:
+
+        ```yaml
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8081
+            initialDelaySeconds: 5
+            periodSeconds: 10
+        ```
+
+  - description: >
+      For Ansible-based operators, the `/ping` endpoint is deprecated. Use `/healthz` and `/readyz` instead of that.
+    kind: "deprecation"
+    breaking: false
+
+  - description: >
+      For Ansible-based operators, the default probe port used was changed from `6789` to `8081`.
+    kind: "change"
+    breaking: false
+    migration:
+      header: (Optional) For Ansible-based operators, the probe port used changed
+      body: >
+        The default probe port used was changed from `6789` to `8081`to keep the project aligned
```

Comment:
Changing the default is a breaking change.

Reply:
I agree, I think changing the port numbers will cause more trouble than it's worth. While consistency across different operator types is nice, they aren't required.